products/AOS-CX-Switching

```diff
@@ -0,0 +1,210 @@
+  template: '..'
+
+
+# A list of releases, supported or not
+# Newer releases go on top of the list, in order
+releases:
+  - releaseCycle: "10.09.xxxx"
+    releaseLabel: ""
+    eol: 2022-12-22
+    support: 2022-12-22
+    release: 2021-12-09
+    latest: "10.09.10"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.08.xxxx"
+    releaseLabel: ""
+    eol: 2022-07-14
+    support: 2022-07-14
+    release: 2021-07-15
+    latest: "10.09.10"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.07.xxxx"
+    releaseLabel: ""
+    eol: 2022-07-15
+    support: 2022-07-15
+    release: 2021-07-15
+    latest: "10.09.10"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.06.xxxx"
+    releaseLabel: ""
+    eol: 2025-11-25
+    support: 2025-11-25
+    release: 2020-11-30
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.05.xxxx"
+    releaseLabel: ""
+    eol: 2021-09-21
+    support: 2021-09-21
+    release: 2020-06-30
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.04.xxxx"
+    releaseLabel: ""
+    eol: 2021-09-21
+    support: 2021-09-21
+    release: 2019-11-1
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: 
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.03.xxxx"
+    releaseLabel: ""
+    eol: 2021-09-21
+    support: 2021-09-21
+    release: 2019-06-19
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+
+# Set an icon for the product from https://simpleicons.org/
+# If the icon is not available on simpleicons, set it to "NA"
+# As an example, https://simpleicons.org/?q=opensuse links to
+# https://simpleicons.org/icons/opensuse.svg and https://simpleicons.org/icons/opensuse.pdf
+# So the slug is `opensuse` (the SVG filename without extension).
+iconSlug: ministryofmagic
+
+# A few extra fields define overall page behaviour
+
+# URL for the page
+permalink: /timeturner
+
+# A list of alternate URLs that will redirect to the permalink. This is nice to let people use easier to remember URLs. For eg, we redirect /golang to /go
+alternate_urls:
+  - /hourglass
```

Comment:
i couldnt see any relation with hourglass url name with the related product so please fix this one or remove